### PR TITLE
SRVCOM-1580: Content reorg, review and editing

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1580

Applies for OCP 4.6+
(might need manual cherry picks due to changes in topicmap)

Direct preview links (most are just content moving places):

- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/discover/knative-event-sources.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/discover/serverless-channels.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-event-sinks.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-apiserversource.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-pingsource.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-kafka-source.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-custom-event-sources.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-creating-channels.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-subs.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-configuring-channels.html
- https://deploy-preview-40461--osdocs.netlify.app/openshift-enterprise/latest/serverless/reference/serverless-listing-event-sources.html (all sections in ref section were just moved, except this one, which has some edits)

### CODE REVIEW

File: serverless/discover/knative-event-sources.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+
+toc::[]
+
+A Knative _event source_ can be any Kubernetes object that generates or imports cloud events, and relays those events to another endpoint, known as a xref:../../serverless/develop/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```
Comment (michaelryanpeter) [inline serverless/discover/knative-event-sources.adoc:9]:
```suggestion
A Knative _event source_ can be any Kubernetes object that generates or imports cloud events and relays those events to another endpoint. The endpoint is known as a xref:../../serverless/develop/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```
Consider rephrasing to keep the sentence under 25 words (though this rephrasing might be a little awkward).

**Review by michaelryanpeter — COMMENTED** [review]
Just a few small comments. Feel free to do what you will the long sentence suggestion.

File: serverless/discover/knative-event-sources.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/discover/knative-event-sources.adoc:9]: I think it reads better as it is, but thanks for the suggestions :)

**Pull request merged**